pkg/models/stats.go

```diff
@@ -51,6 +51,18 @@ type GetDataSourceAccessStatsQuery struct {
 	Result []*DataSourceAccessStats
 }
 
+type ESJSONData struct {
+	ESVersion int
+}
+
+type JSONData struct {
+	JsonData string `json:"jsonData"`
+}
+
+type GetESJSONDataQuery struct {
```

Comment:
I'm not really a big fan of this since it only returns the result from the database. With this you have to unmarschal every row in the usage stats package. I would rather introduce a GetDataSourcesByTypeQuery and implement that in datasource.go together with the rest of data source database commands/queries and returning a slice of `*DataSource` like GetDataSourcesQuery does. WDYT?